docs/blog/newsletter-september-2017.rst

```diff
@@ -0,0 +1,83 @@
+
+The pitfalls of assembling and submitting a docs portfolio are varied. Content can be out of date or apply to outdated technologies. Depending on where the content was published and who by, it might no longer be available online. And if your work was done for internal audiences, it may well be that sharing it would be a breach of your contract. But even with all that, portfolios are often a mandatory part of the interview process.
+
+But just as the community was able to identify the problems with portfolio submissions, they were also able to brainstorm some solutions:
+* Ask your previous employers if they'd be comfortable with you sharing a small snippet of your work.
+* Edit all proprietary names and information out of your content before including it in your portfolio.
+* Consider contributing to an open source project and including those contributions in your portfolio.
+* If you're struggling to find online samples to link to ask if the organziation will accept a file submission instead of an online portfolio.
+* If you're having a hard time finding work that used to be published, use the `Wayback Machine<https://archive.org/web/>`_, or a similar tool.
+
+***********************************
+Resources for Documentation Metrics
+***********************************
+
+If you haven't explored any of the topic-specific channels on Slack, you definitely should. We've got channels for introducing yourself to the community (#intros), getting and giving advice on job-hunting (#career-advice), writing API docs (#documenting-apis), as well as a raft of tool- and location-specific channels. But one of the channels I've been particularly enjoying lately has been the #analytics channel, where folks go to talk about docs metrics!
+
+In the last few weeks, folks have shared some really interesting resources to help us think about how we track and measure the success of our documentation efforts:
+
+* Looking for a place to start with Google Analytics? One of our members - @wouter – shared a dashboard for measuring user documentation in Google Analytics. You can just import it into the property for your docs on Google Analytics and start poking around. It's not a finished product, for sure, but if you're looking for a place to start, this would be a good one. `Download the dashboard here.<https://analytics.google.com/analytics/gallery/#posts/search/%3F_.term%3Duser%20documentation%26_.start%3D0%26_.viewId%3Dja0-XZQsSB-GH7K3Hw3BWw/>`_
```

Comment:
"shared a dashboard for measuring engagement with user documentation ..." ('cuz you're not really measuring the docs. Or "success" instead of "engagement"
Also remove "just" in the following sentence ...

Reply:
👍